proxy/http/HttpConfig.cc

```diff
@@ -809,6 +809,8 @@ register_stat_callbacks()
                      (int)https_incoming_requests_stat, RecRawStatSyncCount);
   RecRegisterRawStat(http_rsb, RECT_PROCESS, "proxy.process.https.total_client_connections", RECD_COUNTER, RECP_PERSISTENT,
                      (int)https_total_client_connections_stat, RecRawStatSyncCount);
+  RecRegisterRawStat(http_rsb, RECT_PROCESS, "proxy.process.http.origin_connections_throttled", RECD_COUNTER, RECP_PERSISTENT,
```

Comment:
It would be better to use connections_throttled_out.  We talked about using this at the standard in identifying connections (_out|_in).